types/koa-csrf/koa-csrf-tests.ts

```diff
@@ -0,0 +1,12 @@
+import * as koaCsrf from 'koa-csrf';
```

Comment:
Since `koaCsrf` is used as a class rather than a module, it should be `import ... = require(...)` here